benchmark/benchmark.gradle

```diff
@@ -22,10 +25,16 @@ jmh {
   fork = 1 // How many times to forks a single benchmark. Use 0 to disable forking altogether
 //  jvmArgs += ["-XX:+UnlockDiagnosticVMOptions", "-XX:+DebugNonSafepoints", "-XX:StartFlightRecording=delay=5s,dumponexit=true,name=jmh-benchmark,filename=$rootDir/benchmark/build/reports/jmh/jmh-benchmark.jfr"]
 //  jvmArgs += ["-agentpath:$rootDir/benchmark/src/jmh/resources/libasyncProfiler.so=start,collapsed,file=$rootDir/benchmark/build/reports/jmh/profiler.txt"]
-  failOnError = true // Should JMH fail immediately if any benchmark had experienced the unrecoverable error?
+
+  // uncomment to use for TypeMatchingBenchmark
```

Comment:
Is it worth having a separate Gradle project?

Reply:
Or separate Gradle task

Reply:
I just tried this, but the jmh task seems different from normal gradle tasks(?): https://github.com/melix/jmh-gradle-plugin/blob/master/src/main/groovy/me/champeau/gradle/JMHTask.java#L32

and so I run into this problem:

https://github.com/melix/jmh-gradle-plugin/issues/161#issuecomment-662367084

Reply:
for now, I modeled it similar to existing benchmarks, and I added #782